shared/common-adapters/avatar.stories.js

```diff
@@ -1,21 +1,24 @@
 // @flow
 import Avatar from './avatar'
 import * as React from 'react'
+import * as Sb from '../stories/storybook'
 import Text from './text'
 import Box from './box'
 import {globalStyles} from '../styles'
-import {storiesOf, action, createPropProvider, PropProviders} from '../stories/storybook'
 
 const sizes = [128, 96, 64, 48, 32, 16]
-const provider = createPropProvider(PropProviders.Avatar(['following', 'both'], ['followers', 'both']))
+const provider = Sb.createPropProviderWithCommon(
```

Comment:
Common was purposefully left out here, although I guess it doesn't matter too much